test/gst-vaapi/decode/hevc.py

```diff
@@ -14,50 +14,16 @@ class default(DecoderTest):
   def before(self):
     # default metric
     self.metric = dict(type = "ssim", miny = 1.0, minu = 1.0, minv = 1.0)
+    self.caps   = platform.get_caps("decode", "hevc_8")
     super(default, self).before()
 
-  @platform_tags(HEVC_DECODE_8BIT_PLATFORMS)
+  @slash.requires(*platform.have_caps("decode", "hevc_8"))
   @slash.requires(*have_gst_element("vaapih265dec"))
-  @slash.parametrize(("case"), sorted([k for k,v in spec.items() if v["width"] <= 4096
-    and v["height"] <= 4096 and v["format"] in mapsubsampling("FORMATS_420")]))
+  @slash.parametrize(("case"), sorted(spec.keys()))
   def test(self, case):
     vars(self).update(spec[case].copy())
     vars(self).update(
       case        = case,
       gstdecoder  = "h265parse ! vaapih265dec",
     )
     self.decode()
-
-  @platform_tags(HEVC_DECODE_8BIT_8K_PLATFORMS)
-  @slash.requires(*have_gst_element("vaapih265dec"))
-  @slash.parametrize(("case"), sorted([k for k,v in spec.items() if (v["width"] > 4096
-    or v["height"] > 4096) and v["format"] in mapsubsampling("FORMATS_420")]))
-  def test_highres(self, case):
-    vars(self).update(spec[case].copy())
-    vars(self).update(
-      case        = case,
-      gstdecoder  = "h265parse ! vaapih265dec",
-    )
-    self.decode()
-
-  @platform_tags(HEVC_DECODE_8BIT_422_PLATFORMS)
-  @slash.requires(*have_gst_element("vaapih265dec"))
-  @slash.parametrize(("case"), sorted([k for k,v in spec.items() if v["format"] in mapsubsampling("FORMATS_422")]))
-  def test_422(self, case):
-    vars(self).update(spec[case].copy())
-    vars(self).update(
-      case        = case,
-      gstdecoder  = "h265parse ! vaapih265dec",
-    )
-    self.decode()
-
-  @platform_tags(HEVC_DECODE_8BIT_444_PLATFORMS)
-  @slash.requires(*have_gst_element("vaapih265dec"))
-  @slash.parametrize(("case"), sorted([k for k,v in spec.items() if v["format"] in mapsubsampling("FORMATS_444")]))
-  def test_444(self, case):
```

Comment:
TODO: add translation for old baseline test name to new test name

Reply:
Delete test_* may cause little difficult to validate only 422/444/8k cases and to count 422/444/8k pass rate in hevc decode logs folder.

Reply:
For now, you can add a prefix to your test case names (e.g. 444_<case name>) in the user config.  Then use the command-line filter option to filter cases with this prefix.  Later, I plan to add a "category" feature to allow us to assign each case a category in the user config.  Categories will improve our reports and let us group together test cases with common attributes.